templates/deployment-web.yaml

```diff
@@ -196,6 +196,20 @@ spec:
             - name: OTEL_SERVICE_NAME_SEPARATOR
               value: "{{ coalesce .Values.mastodon.web.otel.nameSeparator .Values.mastodon.otel.nameSeparator }}"
             {{- end }}
+            {{- if .Values.mastodon.metrics.prometheus.enabled }}
+            - name: MASTODON_PROMETHEUS_EXPORTER_ENABLED
+              value: "true"
+            - name: PROMETHEUS_EXPORTER_HOST
```

Comment:
This should point to the sidecar's host / port, so `127.0.0.1` and the configured port?

Reply:
Those are the defaults for `.Values.prometheus.host|port` (same defaults as in the docs)

Reply:
Yes but there is no point in changing this for users? It will always point to the sidecar container, that listens to 127.0.0.1

Reply:
Fair enough. In that case, we can get rid of `.Values.metrics.prometheus.host` in general, and just have all of them listen on `localhost`.

Reply:
Apologies, poor wording on my part. I currently have it so the sidekiq exporter listens on 0.0.0.0, and the web exporter listens on localhost for the sidecar, and removed the host variable from values.